### DISCUSSION TITLE
Has this project been abandoned?

### DISCUSSION THREAD

**Discussion by chenzanyu** [tracespace/tracespace#429] [Q&A] [answered]
Has this project been abandoned? Will there be an official release of v5?

**Comment by mcous** [comment, marked as answer]
Howdy! I would say the tracespace project is on an ✨indefinite hiatus✨. Been thinking I should update the readme to say as much, so thanks for the poke

- I haven't designed a PCB myself in years, so it's really hard to maintain my interest in the project
- I don't think JS nor SVG are the correct tools for the job of rendering PCBs on the web anymore
- I don't get paid to work on tracespace, nor do I have the energy/desire to seek out such sponsorship

One day in the future, I could return to tracespace to help me learn some new skills, especially if any of the above points change. I think, realistically though, v5 might dead in the water since I've soured on the technical approach
